decidim-core/app/packs/src/decidim/index.js

```diff
@@ -198,6 +199,8 @@ const initializer = (element = document) => {
   // Initialize data-toggles
   element.querySelectorAll("[data-toggle]").forEach((elem) => createToggle(elem))
 
+  element.querySelectorAll("[data-remote-tooltip]").forEach((elem) => fetchRemoteTooltip(elem))
```

Comment:
I think I found a related bug with this approach, but the good news is that we already have it with the `createTooltip` call already 😄 
If you load a proposals index page, and then change the order to any other, the tooltip is not generated again. This can be seen in Metadecidim: first load it gets shown, then you change the order and you don't see the tooltips anymore.

https://meta.decidim.org/processes/roadmap/f/122/proposals

Reply:
This is just a related bug that we can ignore for now. We can work on it in a new issue/PR.

Reply:
Also, this is why @fblupi mentioned in our last meeting something like "the weird thing is that once you load the page and change the order it gets loaded pretty fast"

Reply:
Oh maybe this still doesn't make sense, as `createTooltip` should be 100% client-side 😕

Reply:
The issue with this `bug` is that we  have some js templates that are changing the data, but is not triggering `ajax:loaded` event. I see that we need to fire it in each template , having 2 options: 

#### Custom event 
The same way we do it in `decidim-budgets/app/views/decidim/budgets/projects/index.js.erb`
```
  document.dispatchEvent(new CustomEvent("ajax:loaded", { detail: Javascript_var[0] }));
```

#### Rails UJS way 
```
  Rails.fire(document, "ajax:loaded",  { detail: Javascript_var[0] });
```

Reply:
As I mentioned, I think we can handle that in another PR (and also add proper testing for the tooltip, if we don't have any yet)